### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by AnicetFantomas**
In this pull request we performed : 

- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions and reducers in React.

My presentation link : https://www.loom.com/share/6b3a0f1f4fe2478fa4663b2b22a15ee6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,85 @@
...
 
-### Deployment
+## 📝 License
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This project is [MIT](https://opensource.org/licenses/MIT) licensed.
```
Comment (ahmetbozaci) [inline README.md]: - [ ] Please add a creative common license also. [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.)

File: src/components/Styles/Home.css [hunk 2]
```diff
@@ -1,196 +1,196 @@
...
 
 .logo-container {
-    display: flex;
-    width: 100%;
-    justify-content: space-between;
-    padding: 20px;
-    background-color: #335CFA;
-    height: 50px;
-    font-family: 'Lobster', cursive;
+  display: flex;
```
Comment (ahmetbozaci) [inline src/components/Styles/Home.css]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just an example if you have some CSS rules please combine them also_
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/Styles/Home.css [hunk 3]
```diff
@@ -1,196 +1,196 @@
...
 }
 
 .search-container {
-    display: flex;
-    background-color: #000;
-    justify-content: space-between;
-    padding: 1%; 
-    color: aliceblue;
-    font-size: 25px;
+  display: flex;
```
Comment (ahmetbozaci) [inline src/components/Styles/Home.css]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
This is just an example if you have same CSS rules please combine them.
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/Styles/details.css [hunk 4]
```diff
@@ -1,100 +1,101 @@
...
+  background-size: contain;
 }
 
 .prices {
-    font-size: 25px;
-    margin-top: 30px;
-    display: flex;
+  font-size: 25px;
+  margin-top: 30px;
+  display: flex;
```
Comment (ahmetbozaci) [inline src/components/Styles/details.css]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
This is just an example if you have same CSS rules please combine them.
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/Styles/details.css [hunk 5]
```diff
@@ -1,100 +1,101 @@
...
-    font-weight: bold;
-    color: aliceblue;
-    align-self: center;
-    font-family: 'Open Sans', sans-serif;
+.placeholder,
+.detail-data {
+  font-size: 22px;
+  font-weight: bold;
+  color: aliceblue;
+  align-self: center;
```
Comment (ahmetbozaci) [inline src/components/Styles/details.css:61]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
This is just an example if you have same CSS rules please combine them.
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @AnicetFantomas  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You follow the design guidelines.
- App working as expected.
- You create a presentation video that includes code and demo.
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] You added only snapshot tests Please add unit and integration tests. [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests,the%20Redux%20Store.)
- [ ] It will be better if you move test folder outside the components folder.

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] When you create a presentation video ı recommend using earphones if you didn't use and making sure no background sound. Your voice is coming a little quiet.
- [ ] I think you merged some branches into development branches so that's why ı cant see all files that you created in the files changed section. Please next time make sure don't merge before review.

image

___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @AnicetFantomas , 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [ ] As the previous reviewer suggested, please ensure that there are unit and integration tests as per requirement.

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by CollinsTatang — APPROVED** [review]
Hi @AnicetFantomas , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

image

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**